package.json

```diff
@@ -39,6 +39,7 @@
   "devDependencies": {
     "@remix-run/dev": "^1.4.3",
     "@remix-run/eslint-config": "^1.4.3",
+    "@remix-run/serve": "^1.6.5",
```

Comment:
Required dependency to run `remix dev`

Reply:
I updated this to be the same version as the other Remix dependencies, `^1.4.3`.